src/vms/pvm/txs/fee/complexity.ts

```diff
@@ -227,6 +233,16 @@ export const getBytesComplexity = (
   return result;
 };
 
+export const getWarpComplexity = (message: Bytes): Dimensions => {
+  // TODO: Finish implementation.
```

Comment:
This TODO comes from AvalancheGo. There is no action for us to take on this right now, but obviously warp complexity will be more involved in the future.